scripts/ja/verify-spelling.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+##########
+
+# cd to the root path
+ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/../.." && pwd -P)"
+CONTENT_DIR="content/ja"
+cd "${ROOT}/${CONTENT_DIR}"
+
+TARGET_WORDS=""               # invalid:valid
+TARGET_WORDS="${TARGET_WORDS} コンテナー:コンテナ"
+TARGET_WORDS="${TARGET_WORDS} アーキテクチャー:アーキテクチャ"
+TARGET_WORDS="${TARGET_WORDS} コントローラ[^ー]:コントローラー"
```

Comment:
```suggestion
TARGET_WORDS="${TARGET_WORDS} コントローラ[^ー]:コントローラー"
TARGET_WORDS="${TARGET_WORDS} オペレータ[^ー]:オペレーター"
TARGET_WORDS="${TARGET_WORDS} パラメータ[^ー]:パラメーター"
```

Recently, I reviewed [the PR](https://github.com/kubernetes/website/pull/38185#pullrequestreview-1213656718) what this script should find in.
I would like to add the words `オペレーター` and `パラメーター` to this PR now, because those are commented on [that PR](https://github.com/kubernetes/website/pull/38185#pullrequestreview-1213656718).

Reply:
Thank you, that is a nice point.
I have added those lines.